### PR TITLE
refactor: seperate mvc / handler concerns

### PR DESCRIPTION
**Pull request by ArneD**

### CODE REVIEW

File: src/StreetNameRegistry.Api.Oslo/StreetName/Detail/StreetNameOsloResponse.cs [hunk 1]
```diff
@@ -68,6 +68,9 @@ public class StreetNameOsloResponse
         [JsonProperty(Required = Required.DisallowNull)]
         public StraatnaamStatus StraatnaamStatus { get; set; }
 
+        [JsonIgnore]
+        internal string? LastEventHash { get; set; }
```
Comment (jvandaal) [inline src/StreetNameRegistry.Api.Oslo/StreetName/Detail/StreetNameOsloResponse.cs:72]: Why is this added if it has the JsonIgnore attribute?

File: src/StreetNameRegistry.Api.Oslo/StreetName/List/StreetNameListOsloResponse.cs [hunk 2]
```diff
@@ -34,6 +36,12 @@ public class StreetNameListOsloResponse
         [DataMember(Name = "Volgende", Order = 2, EmitDefaultValue = false)]
         [JsonProperty(Required = Required.Default, DefaultValueHandling = DefaultValueHandling.Ignore)]
         public Uri? Volgende { get; set; }
+
+        [JsonIgnore]
+        internal SortingHeader Sorting { get; set; }
```
Comment (jvandaal) [inline src/StreetNameRegistry.Api.Oslo/StreetName/List/StreetNameListOsloResponse.cs:41]: Why is this added with JsonIgnore?

File: src/StreetNameRegistry.Api.Oslo/StreetName/Detail/StreetNameOsloResponse.cs (hunk 1 above)
Comment (ArneD) [reply src/StreetNameRegistry.Api.Oslo/StreetName/Detail/StreetNameOsloResponse.cs:72]: The controller needs this to add to the etag header.

File: src/StreetNameRegistry.Api.Oslo/StreetName/List/StreetNameListOsloResponse.cs (hunk 2 above)
Comment (ArneD) [reply src/StreetNameRegistry.Api.Oslo/StreetName/List/StreetNameListOsloResponse.cs:41]: The controller needs this to add to the `Response` (HttpResponse).

**Review by jvandaal — APPROVED** [review]
